### DISCUSSION TITLE
Stripping binaries

### DISCUSSION THREAD

**Discussion by ofek** [cross-rs/cross#1258] [Q&A] [answered]
Is there an option to automatically strip the built binaries?

**Comment by Emilgardis** [comment, marked as answer]
Cross can't do this, but cargo can!

You can specify in your Cargo.toml to strip

```
[profile.release]
panic = "abort"
strip = true
```

or pass `-C strip` to rustc

see https://doc.rust-lang.org/cargo/reference/profiles.html#strip

I remember vaguely it not working for every target though as the target might not have strip setup for it

**Reply by ofek** [reply]
Awesome, thanks!
